.mailmap

```diff
@@ -500,9 +508,12 @@ Yuri Weinstein <yuriw@redhat.com> <yuriw@magna002.ceph.redhat.com>
 Yuri Weinstein <yuriw@redhat.com> <yweins@redhat.com>
 Yuri Weinstein <yuriw@redhat.com> <yuri.weinstein@redhat.com>
 Xianxia Xiao <xianxia.xiao@kylin-cloud.com>
+XiaoCheng Wang <wangxiaocheng@sangfor.com.cn>
+Xiaofei Cui <cuixiaofei@sangfor.com.cn> <cuixiaofei@sangfor.com>
 Zack Cerza <zack@redhat.com> <zack.cerza@inktank.com>
 Zack Cerza <zack@redhat.com> <zack@cerza.org>
-Zengran Zhang <zhangzengran@h3c.com>
```

Comment:
i think we should keep as it is. as h3c sponsored the commits contributed by `zhangzengran@h3c.com` when `zhangzengran@h3c.com` was affiliated to H3C.

and the new  and newly added `zhangzengran@sangfor.com.cn` entry will help point this mail address to @zhangsw .